packages/ui/src/app/demos/fib3r/calculator.tsx

```diff
@@ -159,7 +159,7 @@ export class Calculator extends React.Component<
 
   private stopTimer() {
     if (this.interval) {
-      clearInterval(this.interval);
+      window.clearInterval(this.interval);
```

Comment:
`window.` isn't strictly necessary. Is there a reason  you prefer it? Is it just to be explicit?

Reply:
Ya, because either we have to remove the node types, which means require will be missing or we do the window.

Reply:
I mean for all of these, they can be called without `window.`. I understand why the `NodeJS.` type is a problem (browser projects won't be set to provide that type) but you should be able to call these functions just fine without referencing the global window object:

![image](https://github.com/open-feature/playground/assets/25272906/bfacf4df-8bbd-4059-975e-56d4ca793cce)

Reply:
Yes but at least for me, tsc will not accept it without window because node types are there and take precendence. 
When using window, the DOM types are used. Without window I could not assign setIntetval to a number.
It works when removing the node types, but then we loose the type for require.

Reply:
Ah, OK. That makes sense. I guess it has something to do with the project setup, but this is good for now.

Reply:
Yes it has, maybe we can see if we can remove the node types from the frontend later.